frontend/javascripts/oxalis/view/left-border-tabs/controls_and_rendering_settings_tab.js

```diff
@@ -91,20 +102,32 @@ class ControlsAndRenderingSettingsTab extends PureComponent<ControlsAndRendering
         return (
           <Panel header="Viewport Options" key="2">
             <LogSliderSetting
-              label={settingsLabels.zoom}
+              label={
+                <Tooltip title="Zoom in or out in the data viewports">
+                  {settingsLabels.zoom}
+                </Tooltip>
+              }
               roundTo={3}
               min={this.props.validZoomRange[0]}
               max={this.props.validZoomRange[1]}
               value={this.props.zoomStep}
               onChange={this.props.onChangeZoomStep}
             />
             <SwitchSetting
-              label={settingsLabels.displayCrosshair}
+              label={
+                <Tooltip title="Show crosshair lines in each viewport, with colors indicating the other viewports">
```

Comment:
some of the tooltip texts (such as this one) are duplicated. can you please extract the strings into a variable (e.g., something like `settingsLabels.displayCrosshairTooltip` in `messages.js`? alternatively, you could try to extract the entire `<SwitchSetting />` node into a method-local variable. The components look very similar and if they are equal we should remove the redundancy on that level.

Reply:
I extracted the tooltip strings to messages.js. To be honest I am not sure what the difference between the case `Constants.MODE_PLANE_TRACING` and case `Constants.MODE_VOLUME` is at all. Also `default` seems to be flight mode? Maybe this could be replaced by a simple if-else?

Reply:
yeah, this seems to be the result of historic growth (from a time where the skeleton options where in the same tab). thanks for taking the time to extract the tooltip messages :+1: I think, we can leave the PR now as is :)

Reply:
I replaced the switch by if (arbitrary or flight) {flight settings} else {orthogonal settings}, could you have another look? Or would you have preferred the old version?

Reply:
awesome! looks perfect :+1: